sql/src/main/antlr/OpenSearchSQLParser.g4

```diff
@@ -414,7 +431,7 @@ legacyRelevanceFunctionName
     ;
 
 functionArgs
-    : functionArg (COMMA functionArg)*
+    : (functionArg (COMMA functionArg)*)?
```

Comment:
Is this change necessary? Because I see all others use `functionArgs?` already.

Reply:
The goal is to have the same behavior in SQL and PPL. This fix makes `FunctionArgsContext` always not null. In case if there is no args, it contains an empty list.
See changes in `AstExpressionBuilder`s: https://github.com/Bit-Quill/opensearch-project-sql/blob/integ-datetime-now/ppl/src/main/java/org/opensearch/sql/ppl/parser/AstExpressionBuilder.java#L272

Reply:
Okay. My question was with this change, is all question mark still required, one example: https://github.com/Bit-Quill/opensearch-project-sql/blob/integ-datetime-now/sql/src/main/antlr/OpenSearchSQLParser.g4#L303

Reply:
Thank you, I removed them from SQL, PPL already don't have them. See fix in bd79364b.